go-controller/pkg/ovn/egressfirewall_dns.go

```diff
@@ -148,9 +158,9 @@ func (e *EgressDNS) addToDNS(dnsName string) {
 // EgressFirewalls. The loop runs after receiving one of two signals
 // 1. time.After(durationTillNextQuery) times out and the dnsName with the lowest ttl is checked
 //    and the durationTillNextQuery is updated
-// 2. e.added is recived and durationTillNextQuery is recomputed
+// 2. e.added is received and durationTillNextQuery is recomputed
```

Comment:
nit: could you update this comment to refer to the three signals?

Reply:
Done!